<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintenanceWidget</class>
 <widget class="QWidget" name="MaintenanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:1px;
border-radius:5px</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>220</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="CloseButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>90</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Yu Gothic</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border:1px solid #CCCCCC;
	border-radius:3px;
	color:#666;
}
QPushButton:hover
{
	background-color:#f8f8f8;
	border:1px solid #bfbfbf;
	border-radius:3px;
	color:#666;
}
QPushButton:pressed
{
	background-color:#f0f0f0;
	border:1px solid #bfbfbf;
	border-radius:3px;
	color:#666;
}</string>
         </property>
         <property name="text">
          <string>close</string>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <zorder>CloseButton</zorder>
        <zorder>CloseButton</zorder>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>160</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="MoveLeftButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>55</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:#0D47A1;
	border-width:1px;
	border-radius:25px;
	color:white;
}
QPushButton:hover
{
	background-color:#0B3E8C;
	border-width:1px;
	border-radius:25px;
	color:white;
}
QPushButton:pressed
{
	background-color:#0A3880;
	border-width:1px;
	border-radius:25px;
	color:white;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/ic_arrow_back_white_24dp.png</normaloff>:/uiIcon/src/ui/resource/icon/ic_arrow_back_white_24dp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="MoveUpButton">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>0</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:#0D47A1;
	border-width:1px;
	border-radius:25px;
	color:white;
}
QPushButton:hover
{
	background-color:#0B3E8C;
	border-width:1px;
	border-radius:25px;
	color:white;
}
QPushButton:pressed
{
	background-color:#0A3880;
	border-width:1px;
	border-radius:25px;
	color:white;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/ic_arrow_upward_white_24dp.png</normaloff>:/uiIcon/src/ui/resource/icon/ic_arrow_upward_white_24dp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="MoveRightButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>55</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:#0D47A1;
	border-width:1px;
	border-radius:25px;
	color:white;
}
QPushButton:hover
{
	background-color:#0B3E8C;
	border-width:1px;
	border-radius:25px;
	color:white;
}
QPushButton:pressed
{
	background-color:#0A3880;
	border-width:1px;
	border-radius:25px;
	color:white;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/ic_arrow_forward_white_24dp.png</normaloff>:/uiIcon/src/ui/resource/icon/ic_arrow_forward_white_24dp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="MoveDownButton">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>110</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	background-color:#0D47A1;
	border-width:1px;
	border-radius:25px;
	color:white;
}
QPushButton:hover
{
	background-color:#0B3E8C;
	border-width:1px;
	border-radius:25px;
	color:white;
}
QPushButton:pressed
{
	background-color:#0A3880;
	border-width:1px;
	border-radius:25px;
	color:white;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../rc.qrc">
           <normaloff>:/uiIcon/src/ui/resource/icon/ic_arrow_downward_white_24dp.png</normaloff>:/uiIcon/src/ui/resource/icon/ic_arrow_downward_white_24dp.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>220</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../rc.qrc"/>
 </resources>
 <connections/>
</ui>
